lstchain/reco/utils.py

```diff
@@ -455,6 +454,8 @@ def filter_events(events,
     for k in filters.keys():
         if k in events.columns:
             filter &= (events[k] >= filters[k][0]) & (events[k] <= filters[k][1])
+        else:
+            raise TypeError(f'{k} does not exist in the list of keys')
```

Comment:
- That would be a `KeyError`
- do we want an error or a warning here?

Reply:
error, always.

nobody reads warnings in cluster job logs.

Reply:
In this case, there is no need for an `if - else` condition anymore.

Reply:
ok, you're right. I removed `if-else` condition